src/client/app/containers/editor.ts

```diff
@@ -39,6 +39,16 @@ export class Editor implements AfterViewInit {
      */
     async ngAfterViewInit() {
         this._monacoEditor = await this._monaco.create(this._editor, { theme: 'vs' });
+        let editor = this._monacoEditor;
```

Comment:
Oh, I see, interesting.  That might be worth pointing out in a comment.  And in that case I would initially do a `const editor = await this._monaco.create(...)`, and then set `this._monacoEditor = editor` at the very end.